src/lib/htmlInputPropsUtils.js

```diff
@@ -37,15 +37,22 @@ export const htmlInputProps = [...htmlInputAttrs, ...htmlInputEvents]
 /**
  * Returns an array of objects consisting of: props of html input element and rest.
  * @param {object} props A ReactElement props object
- * @param {array} [htmlProps] An array of html input props
+ * @param {Object} [options={}]
+ * @param {Array} [options.htmlProps] An array of html input props
+ * @param {boolean} [options.includeAria] Includes all input props that starts with "aria-"
  * @returns {[{}, {}]} An array of objects
  */
-export const partitionHTMLInputProps = (props, htmlProps = htmlInputProps) => {
+export const partitionHTMLInputProps = (props, options = {}) => {
+  const {
+    htmlProps = htmlInputProps,
+    includeAria = true,
+  } = options
```

Comment:
I've applied there our `options` pattern, so we can control `htmlProps` and `ariaProps` separately.